gradle.properties

```diff
@@ -1,4 +1,7 @@
 # Sets default memory used for gradle commands. Can be overridden by user or command line properties.
 # This is required to provide enough memory for the Minecraft decompilation process.
 org.gradle.jvmargs=-Xmx3G
-org.gradle.daemon=false
\ No newline at end of file
+org.gradle.daemon=false
+#add curio version variable
+curios_version=5.12.0
```

Comment:
I would prefer to have the values in the build.gradle,
I don't like to have things in different files.